ci-operator/step-registry/telcov10n/functional/hcp-cnf/tests/telcov10n-functional-hcp-cnf-tests-commands.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "Hello from compute QE CLUSTER!"
```

Comment:
We first need to implement some logic here, there is no point merging it without logic.
If you need to test that it's working, rehearse it.

Reply:
Okay then I'll entirely remove it now and from the workflow itself, will add them in a later PR